00new_packages.autobuild

```diff
@@ -1,4 +1,7 @@
 # IMPORTANT: new packages must be added at the top of this file
 # They must NEVER be added to other files
 
-
+cmake_package 'gui/rock_webapp'
+ruby_package 'tools/rest_api' do |pkg|
+    Autoproj.env_add_path 'ROCK_WEBAPP', File.join(pkg.prefix, 'share', 'rest_api')
```

Comment:
Since it is a path, you basically prepend the "test environment" at the front. This way, you get to load "test fixtures" first and fallback to global ones otherwise.

Given that it is how all the path stuff works (e.g. RUBYLIB or PYTHONPATH or KDEPREFIXES), I guess that it should not be completely broken.